denops_std/function/cursor.ts

```diff
@@ -118,17 +121,27 @@ export async function virtcol(
  * then applies to another buffer.
  * To get the column number use |col()|.  To get both use
  * |getpos()|.
+ * With the optional {winid} argument the values are obtained for
+ * that window instead of the current window.
+ * Returns 0 for invalid values of {expr} and {winid}.
  * Examples:
  * 	line(".")		line number of the cursor
+ * 	line(".", winid)	idem, in window "winid"
  * 	line("'t")		line number of mark t
- * 	line("'" . marker)	line number of mark marker
+ * 	line("'" .. marker)	line number of mark marker
+ *
  * To jump to the last known position when opening a file see
  * |last-position-jump|.
+ *
  * Can also be used as a |method|:
  * 	GetValue()->line()
  */
-export async function line(denops: Denops, expr: string): Promise<number> {
-  return await denops.call("line", expr) as number;
+export async function line(
+  denops: Denops,
+  expr: string,
+  winid?: number | string,
```

Comment:
Do we really need string in signatures?
```suggestion
  winid?: number,
```

It's out of the scope of change, but `screenpos()` has same issue.

Reply:
Yes, it is needed.
I added unit test for `string`.

Reply:
`screenpos()` already has unit test for `string`

Reply:
Removed `string` signatures from `line()`, `screenpos()`, `getcurpos()`.

Reply:
Sorry, I'm mistake.
`winid` received a number or a string like `"%"`.
We should specify `"%"` instead of `"1000"` in tests.

Reply:
Fixed.